_drafts/2020-07-27-data_science_plotting_notebook_template.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: "Jupyter Notebook Templates for Data Science: Plotting"
+description: >
+  Jupyter notebooks are great for data exploration; jumpstart your
```

Comment:
Feels like the first chunk of this only exists to get the Jupyter buzzword in there. Maybe

Jumpstart your visualizations with this Jupyter plotting notebook!